assets/sass/okta-theme.scss

```diff
@@ -756,6 +756,7 @@
     border-color: $disabled-primary-button-bg-color;
     background-image: none;
     box-shadow: none;
+    opacity: 0.5;
```

Comment:
seems like there may be a small difference here. The "opacity" rule added by the "template" function is only applied if `colors.brand` is set, but the CSS rule here would apply in all cases.